src/components/timeline/ActiveTabGlobalTrack.js

```diff
@@ -17,10 +17,10 @@ import {
   getActiveTabResourceTracks,
 } from 'firefox-profiler/selectors/profile';
 import './Track.css';
-import TimelineTrackThread from './TrackThread';
-import TimelineTrackScreenshots from './TrackScreenshots';
-import ActiveTabResourcesPanel from './ActiveTabResourcesPanel';
-import { assertExhaustiveCheck } from 'firefox-profiler/utils/flow';
+import { TrackThread } from './TrackThread';
+import { TrackScreenshots } from './TrackScreenshots';
+import { ActiveTabResourcesPanel } from './ActiveTabResourcesPanel';
+import { assertExhaustiveCheck } from '../../utils/flow';
```

Comment:
I see they were places where we used `firefox-profiler` that you changed to `../..` (or similar patterns), can you please put them back? I'm not asking to change all of them, just the ones you're changing in this patch. I believe this comes from a bad conflict resolution.

**Note:** the ones with `./` are fine.